packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -112,7 +110,6 @@ export type TApplicationContext<AdditionalEnvironmentProperties extends {}> = {
   project: ReturnType<typeof mapProjectToApplicationContextProject>;
   permissions: TApplicationContextPermissions | null;
   actionRights: TApplicationContextActionRights | null;
-  menuVisibilities: TApplicationContextMenuVisibilities | null;
```

Comment:
This now is not exposed anymore in the application context, as it's only used by the navbar anyway.

It's a breaking change, but more of an internal one.